src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -83,4 +83,13 @@ default String getAsTextHidePrivate() {
         }
         return builder.toString();
     }
+    
+    default String getPrintableString(Printable... printables){
+    	String value = "";
+    	for(Printable printable : printables){
```

Comment:
Good job using the for each loop. Remember that you can also use StringBuilder, StringJoiner, and String.join to perform string appends.